src/pages/[productSlug]/integrations/index.tsx

```diff
@@ -48,7 +52,9 @@ export function generateProductIntegrationLibrarySidebarNavData(
 	}
 }
 
-export async function getServerSideProps({ params }) {
+export async function getServerSideProps({
+	params,
+}): Promise<GetStaticPropsResult<ViewProps & { metadata: HeadMetadataProps }>> {
```

Comment:
One tangential change in this PR was adding a return type here to the props. Felt like the right thing to do given I was adding in a `productSlug` property to the returned props 👍